sky/packages/sky/lib/widgets/scrollable.dart

```diff
@@ -370,7 +382,7 @@ abstract class FixedHeightScrollable extends Scrollable {
     return new SizeObserver(
       callback: _handleSizeChanged,
       child: new Viewport(
-        offset: offsetY,
+        scrollOffset: new Offset(0.0, offsetY),
```

Comment:
I guess Fixed and VariableHeightScrollable will eventually need to support the vertical and horizontal ViewportScrollDirections.